### ISSUE TITLE
Identifo Admin: Key is invalid error when trying to login

### ISSUE THREAD

**Issue by kazemisoroush** [MadAppGang/identifo#216]
This issue is happening on `v2.1.2`
The public and private key pair is identified and showed in the admin panel but when I try to login I'm prompted with an error. Please find more information in this video:
https://www.loom.com/share/61f305574b374f3eb4a05c101fcb2986

```
2021/09/16 00:12:57 s3 config poll watcher checking the config file ...
API_ROUTER: 2021/09/16 00:12:59 router.go:152: api error: api.internal_server_error (status=500). Details: key is invalid. Where: LoginWithPassword.LoginFlowError.
[negroni] 2021-09-16T00:12:59Z | 500 |   103.344744ms | identifo.dev.evergen.technology | POST /auth/login
A
```

**Comment by kazemisoroush** [comment]
![image](https://user-images.githubusercontent.com/8931595/133529393-cb060627-2959-4383-ac45-ddefd013b17c.png)
![image](https://user-images.githubusercontent.com/8931595/133529412-2c8b5743-e041-4123-a8b7-55d3bd8df110.png)

**Comment by erudenko** [comment]
Fixed in `v2.1.3`
